workspace.json

```
@@ -740,6 +740,13 @@
       "tags": ["vNext", "platform:web"],
       "implicitDependencies": []
     },
+    "@fluentui/react-storybook-addon-codesandbox": {
+      "root": "packages/react-components/react-storybook-addon-codesandbox",
+      "projectType": "library",
+      "implicitDependencies": [],
+      "sourceRoot": "packages/react-components/react-storybook-addon-codesandbox/src",
+      "tags": ["vNext", "platform:node"]
```

`react-storybook-addon` uses web, wdyt?

```suggestion
      "tags": ["vNext", "platform:web"]
```